CourseManager/Courses/Course 4 UI and Animations in SwiftUI/Views/Detail/WeatherView.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  WeatherView.swift
+//  CourseManager
+//
+//  Created by Summer Crow on 2022-11-10.
+//
+
+import SwiftUI
+
+struct WeatherView: View {
+    @State private var searchText = ""
+    
+    var searchResults: [Forecast] {
+        if searchText.isEmpty {
+            return Forecast.cities
+        } else {
```

Comment:
else here is extra, also ternary if statement looks shorter

Reply:
Updated to the following: 
  var searchResults: [Forecast] {
        return searchText.isEmpty ? Forecast.cities : Forecast.cities.filter { $0.location.contains(searchText)
        }
    }